components/LocalFooter/LocalFooter.tsx

```diff
@@ -9,104 +9,94 @@ import { getProcessedImage } from '@/utilities/getProcessedImage';
 import { initiatives, themes } from '@/utilities/routes';
 
 export const LocalFooter = () => (
-  <>
-    <Image
-      width={2000}
-      height={40}
-      alt=""
-      loading="lazy"
-      src={getProcessedImage('https://a-us.storyblok.com/f/1005200/2000x40/f23b53c0e4/steve-johnson-cropped-2000x40-02.jpg') || ''}
-      className="w-full"
-    />
```

Comment:
Just removing the hard coded textured horizontal bar in the local footer. I'm going to turn this into a Storyblok component so different bars can be picked.